incubator/vault-token-injector/values.yaml

```diff
@@ -25,7 +48,7 @@ config:
     env_variable: VAULT_TOKEN
 
 metrics:
-  # -- If true, a prometheus endpoint will be enabled on port 4329
+  # -- If true, a prometheus endpoint will be enabled on port 4329. This only applies to the deployment
```

Comment:
Maybe just put this under `deployment` then? Not a blocker.

Reply:
I thought about that, and originally was trying not to introduce breaking changes, but then I did anyway. Will update.